src/view/actions/resetEventMergeId.jsx

```diff
@@ -12,47 +12,56 @@ governing permissions and limitations under the License.
 
 import "regenerator-runtime"; // needed for some of react-spectrum
 import React from "react";
-import Select from "@react/react-spectrum/Select";
+import { object, string } from "yup";
+import Textfield from "@react/react-spectrum/Textfield";
 import FieldLabel from "@react/react-spectrum/FieldLabel";
 import "@react/react-spectrum/Form"; // needed for spectrum form styles
 import render from "../render";
 import WrappedField from "../components/wrappedField";
 import ExtensionView from "../components/extensionView";
-import getInstanceOptions from "../utils/getInstanceOptions";
-import "./instanceNameOnly.styl";
+import singleDataElementRegex from "../constants/singleDataElementRegex";
+import InfoTipLayout from "../components/infoTipLayout";
+import "./resetEventMergeId.styl";
 
 const getInitialValues = ({ initInfo }) => {
-  const { instanceName = initInfo.extensionSettings.instances[0].name } =
-    initInfo.settings || {};
+  const { eventMergeId = "" } = initInfo.settings || {};
 
   return {
-    instanceName
+    eventMergeId
   };
 };
 
 const getSettings = ({ values }) => {
-  return {
-    instanceName: values.instanceName
-  };
+  return values;
 };
 
-const InstanceNameOnly = () => {
+const validationSchema = object().shape({
+  eventMergeId: string().matches(
+    singleDataElementRegex,
+    "Please specify a data element"
+  )
+});
+
+const ResetEventMergeId = () => {
   return (
     <ExtensionView
       getInitialValues={getInitialValues}
       getSettings={getSettings}
-      render={({ initInfo }) => {
+      validationSchema={validationSchema}
+      render={() => {
         return (
           <div>
-            <FieldLabel labelFor="instanceNameField" label="Instance" />
+            <InfoTipLayout tip="Please specify the data element that represents the event merge ID you would like to reset.">
+              <FieldLabel labelFor="eventMergeIdField" label="Event Merge ID" />
+            </InfoTipLayout>
             <div>
               <WrappedField
-                data-test-id="instanceNameField"
-                id="instanceNameField"
-                name="instanceName"
-                component={Select}
+                data-test-id="eventMergeIdField"
+                id="eventMergeIdField"
+                name="eventMergeId"
+                component={Textfield}
                 componentClassName="u-fieldLong"
-                options={getInstanceOptions(initInfo)}
+                supportDataElement="replace"
```

Comment:
Notice we do _not_ ask for an instance on this action. That's because there's really no need to, since the cache is not specific to any particular instance.